### PR TITLE
Update depending react-on-rails to ^10.0.0

### PR DESCRIPTION
**Pull request by tricknotes**
But I'm not sure why the dependency is required...

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/959)

### CODE REVIEW

**Comment by shepmaster** [comment]
> But I'm not sure why the dependency is required...

I'd also like to know why the package depends on itself; I can't quite fathom why that would be useful 😸

File: yarn.lock [hunk 1]
```diff
@@ -3248,6 +3241,13 @@ react-dom@^15.6.1:
...
     prop-types "^15.5.10"
 
+react-on-rails@^10.0.0:
+  version "10.0.0"
+  resolved "https://registry.yarnpkg.com/react-on-rails/-/react-on-rails-10.0.0.tgz#afc658bf23f5dfe3fd836adc178eb4672727a89d"
+  dependencies:
+    react-on-rails "^9.0.3"
+    react-redux "^5.0.6"
+
 react-on-rails@^9.0.0-beta.12:
```
Comment (shepmaster) [inline yarn.lock:3251]: Should this old version still be here?

**Comment by justin808** [comment]
@tricknotes @shepmaster Thank for this one. However, @Judahmeek will be submitting a fix soon that has this.

**Comment by tricknotes** [comment]
hi, @Judahmeek.
How do you have any plan for this fix?
Do you mind if I try to remove this dependency?

**Comment by tricknotes** [comment]
I send a new PR to remove these dependencies https://github.com/shakacode/react_on_rails/pull/968

**Pull request closed without merging**